source/testnet/guides/become-baker.rst

```diff
@@ -279,25 +261,57 @@ To update the baker stake run
 
-When a baker adds stake for the first time or increases their stake:
+   +----------------------------------------+------------------------------------+----------------+
+   |                                        | Transaction is included in a block | After 2 epochs |
+   +========================================+====================================+================+
+   | Change is visible by querying the node | ✓                                  |                |
+   +----------------------------------------+------------------------------------+----------------+
+   | Baker uses the new stake               |                                    | ✓              |
+   +----------------------------------------+------------------------------------+----------------+
 
-- That change becomes visible on the chain immediately (e.g. through ``concordium-client account show bakerAccount``)
-- The baker can bake (and possibly finalize, if stake is sufficient) after 2 epochs
+When a baker **decreases the staked amount**, the change will need *2 +
+bakerCooldownEpochs* epochs to take effect. The change becomes visible on the
+chain as soon as the transaction is included in a block, it can be consulted through
+``concordium-client account show bakerAccount``:
 
-When a baker decreases the stake amount:
+.. code-block:: console
+
+   $concordium-client account show bakerAccount
+   ...
+
+   Baker: #22
+    - Staked amount: 50.000000 GTU to be updated to 20.000000 GTU at epoch 261  (2020-12-24 12:56:26 UTC)
+    - Restake earnings: yes
 
-- The change becomes visible on the chain after the *cooldown period* (currently 168 epochs)
+   ...
 
-  * - The pending change can also be queried immediately after the  using
-      ``concordium-client raw GetAccountInfo`` and observing the ``pendingChange`` attribute
+.. table:: Time constraints when decreasing the stake
 
-- The change takes effect after the cooldown period plus 2 epochs
+   +----------------------------------------+------------------------------------+----------------------------------------+
+   |                                        | Transaction is included in a block | After *2 + bakerCooldownEpochs* epochs |
+   +========================================+====================================+========================================+
+   | Change is visible by querying the node | ✓                                  |                                        |
+   +----------------------------------------+------------------------------------+----------------------------------------+
+   | Baker uses the new stake               |                                    | ✓                                      |
+   +----------------------------------------+------------------------------------+----------------------------------------+
+   | Stake can be decreased again or        | ✗                                  | ✓                                      |
```

Comment:
What does the ✗ mean here? Should this be just empty?

Reply:
This means that it is forbidden to perform that operation at that time. This one is more of a restriction than when will something take effect, that's why it has an X.

Reply:
I see, OK.